docs/docs/articles/step2-installing-agent.md

```diff
@@ -0,0 +1,51 @@
+# Step 2 - Installing the Testkube Agent
```

Comment:
the instruction here is for creating another environment, but free accounts can only have 1 env max: https://testkube.io/pricing

Can you please do the instruction for when someone is creating a new account completely (to simulate that, just create a new org and you'll have the welcome page).